css/admin.css

```diff
@@ -326,4 +326,22 @@ div[data-name^=oopluginfieldconfigformdefaultsvalues-value].multiselect > input.
 	color: #646970;
 	font-style: italic;
 	text-decoration: none;
+}
+input#onoffice-input-submit {
+	color: #2271b1;
+	border-color: #2271b1;
+	border-style: solid;
+	border-radius: 2px;
+	border-width: 1px;
+}
+
+body #post-body-content .viewusage, body #post-body-content .wp-clearfix{
+	display: inline-block;
+}
+body #post-body-content p.wp-clearfix:nth-child(1){
+	display: block !important;
+}
+#button-copy{
+	margin-left: 5px;
+	font-size: 1.2em;
```

Comment:
Also, looking at the [video](https://github.com/onOfficeGmbH/oo-wp-plugin/pull/160#issuecomment-1135767892), the button for copying looks pretty large. It should be the same size as the other WordPress-buttons.

Could this be because of this `font-size`?